create_per_gene_matrix.py

```diff
@@ -440,26 +439,34 @@ def compute_from_full_mt(test: bool, overwrite: bool) -> None:
     )
 
     logger.info(
-        "Annotating full MatrixTable with VEP, filter information, and all relevant allele frequencies "
-        "(variant popmax AF <= AF cutoff for the following cutoffs: %s)...",
+        "Annotating full MatrixTable with VEP, filter information, annotation indicating all relevant consequence "
+        "groupings for the variant (matching the annotation used in compute_from_vp_mt), "
+        "bottlenecked population AF (fin, asj, oth), and all relevant allele frequencies for filtering "
+        "(variant popmax or global AF <= AF cutoff for the following cutoffs: %s)...",
         ALLELE_FREQUENCY_CUTOFFS,
     )
     mt = mt.select_rows(
         vep=vep_expr.values().map(lambda k: k.annotate(csq=cum_csq_map.get(k.csq))),
         filters=rf_ht[mt.row_key].filters,
-        af_cutoff=af_groups.filter(lambda af: freq_ht[mt.row_key].popmax[0].AF <= af),
+        af_cutoff=af_groups.filter(lambda af: hl.max(freq_ht[mt.row_key].popmax[0].AF,freq_ht[mt.row_key].freq[0].AF, filter_missing=True) <= af),
+        bottlenecked_af=hl.max(freq_ht[mt.row_key].freq[3].AF, freq_ht[mt.row_key].freq[4].AF, freq_ht[mt.row_key].freq[5].AF, filter_missing=True),
```

Comment:
use freq_meta and the bottlenecked pop global to get this max

Reply:
done, thank you!